pkg/webhook/request.go

```diff
@@ -85,23 +85,27 @@ func (s *webhookServer) serve(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var admissionResponse *admissionv1beta1.AdmissionResponse
-	ar := admissionv1beta1.AdmissionReview{}
+	var admissionResponse *admissionv1.AdmissionResponse
+	ar := admissionv1.AdmissionReview{}
 	if _, _, err := s.deserializer.Decode(body, nil, &ar); err != nil {
 		log.Error(err, "Can't decode body")
 		admissionResponse = notAllowed(err.Error(), metav1.StatusReasonBadRequest)
 	} else {
 		admissionResponse = s.validate(&ar)
 	}
 
-	admissionReview := admissionv1beta1.AdmissionReview{}
+	admissionReview := admissionv1.AdmissionReview{}
 	if admissionResponse != nil {
 		admissionReview.Response = admissionResponse
 		if ar.Request != nil {
 			admissionReview.Response.UID = ar.Request.UID
 		}
 	}
 
+	// APIVersion and Kind must be set for admission/v1, or the request would fail
+	admissionReview.APIVersion = "admission.k8s.io/v1"
```

Comment:
Small nit. The `admissionReview`can be initialized APIVersion and Kind at creation created:

```
admissionReview := admissionv1.AdmissionReview{
  APIVersion: admissionv1.SchemeGroupVersion.String(),
  Kind: "AdmissionReview"
}
```

Reply:
Ahh good point!